libs/wingsdk/test/target-tf-aws/api.test.ts

```diff
@@ -68,7 +68,7 @@ test("api with GET routes with common prefix", () => {
 
   // THEN
   expect(tfResourcesOfCount(output, "aws_api_gateway_rest_api")).toEqual(1);
-  expect(tfResourcesOfCount(output, "aws_lambda_function")).toEqual(2);
+  expect(tfResourcesOfCount(output, "aws_lambda_function")).toEqual(1);
```

Comment:
API previously deduped inflights only if there were the exact same instance (referentially)

With _hash, we can understand when two inflight are equal in a non-referential way. So now this test actually dedupes these functions.

Reply:
are you sure the hash is safe vis-a-vis captures? can we really assume that two inflight closures are the same just based on their code?

Reply:
The "code" includes captures, so yes it should be safe